dashboard/lib/pd/foorm/legacy_survey_summaries.rb

```diff
@@ -0,0 +1,52 @@
+# Retrieves, parses and summarizes Foorm Survey results for consumption by APIs.
+module Pd::Foorm
+  class LegacySurveySummaries
+    def self.get_summaries(facilitator)
+      data = Pd::LegacySurveySummary.where(
```

Comment:
nit: add comments before each of these calls to describe what data they are getting